tests/baselines/reference/user/eventemitter3.log

```diff
@@ -0,0 +1,18 @@
+Exit Code: 1
+Standard output:
+node_modules/eventemitter3/index.d.ts(22,13): error TS2344: Type 'T[K]' does not satisfy the constraint 'any[] | ((...args: any[]) => void)'.
```

Comment:
Error is different Monday morning than Friday afternoon, and eventemitter3 has published 4.0.1-.3 in the last 24 hours, so I think they're trying to fix type errors manually over there.

Reply:
"Better TypeScript type definitions": https://github.com/primus/eventemitter3/issues/217

They will get it figured out eventually I guess.